src/Environment.js

```diff
@@ -11,7 +11,7 @@ import {
   // Define a function that fetches the results of an operation (query/mutation/etc)
   // and returns its results as a Promise:
   function fetchQuery(operation, variables, cacheConfig, uploadables) {
-    return fetch('http://localhost:4001', {
+    return fetch('https://chuck-say-api-dwwevjkboq.now.sh/', {
```

Comment:
nice one
we can configure an `alias` later on

Reply:
For sure